src/pineko/cli/check_scalevar.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
let's move this to the cli/check module and rename the other function in there accordingly ...